docs/db/configuration.md

```diff
@@ -1,8 +1,8 @@
 import Issues from '../getting-started/issues.md';
 
-# Configuration 
+# Configuration
```

Comment:
Why was this file changed at all?

Reply:
I changed examples to use Valkey instead of Redis.
I thought you wanted me to change occurences of Redis everywhere. Am I wrong?

Reply:
It was autoformat's fault. I reverted it and reapplied only required changes. How does it look now?